src/config/urls.py

```diff
@@ -11,7 +11,8 @@
 )
 
 urlpatterns = [
-    path("api/v1/", include("safe_apps.urls", namespace="v1")),
+    path("api/v1/", include("safe_apps.urls", namespace="safe-apps")),
```

Comment:
what does the `namespace` do?

Reply:
This is mostly useful for the `reverse()` resolution of the url. So even if different included apps have the same pattern in the url (imagine something like `/index`) we can distinguish them by using `reverse()` with the `namespace`. Reverse is also to avoid using full paths – if the path changes, the url is correctly resolved to the view.

I've tried having everything under the `v1` namespace (left a comment about it here: https://github.com/gnosis/safe-config-service/pull/88#discussion_r643097398) but probably doesn't make too much sense to do it that way.